### ISSUE TITLE
Yahya Bot (Legendary Island)

### ISSUE THREAD

**Issue by TheOGPickle** [Wynncraft/Issues#8366]
So when reaching the Yahya Bot in Legendary Island, there's been a little problem. This stage requires you to stay alive long enough until the door opens/Yahya kills himself. However, after afking for hours on end, Yahyas not dead and the door is not open. Not sure if this is a legitimate problem or it's just being an idiot, but any help is appreciated.

**Comment by thecoolkogut4** [comment]
Your meant to kill yayah. You walk through and wait for 30 seconds then get tped, wait another 30, wait 1min, 1min, 1.5, 1.5 then you should be back in a room that looks like the first one. Yayah should be in the center and then you need to kill him. If you loged out in the middle of getting tped I believe it breaks and you need to /kill to get out. What does the room that your in look like? Is yayah there? What happens when you try and hit it? How much health does yayah have? Is he attacking you?

**Comment by TheOGPickle** [comment]
When I first entered the room, I first attacked Yahya who had 1000000 hp (maybe shouldn't have) but I waited far longer than 30 seconds, about... 4 minutes. I didn't get tped in those 4 minutes, so I did /class to see if anything was fixed (which probably broke it). To answer the rest of your questions, the room was how it normally looks first going in, Yahya is there, nothing happens when I hit him, and he attacks me.
